cobalt/black_box_tests/black_box_tests.py

```diff
@@ -122,15 +131,28 @@ def tearDownClass(cls):
     super(BlackBoxTestCase, cls).tearDownClass()
     logging.info('Done %s', cls.__name__)
 
-  def CreateCobaltRunner(self, url=None, target_params=None):
+  def CreateCobaltRunner(self,
+                         url=None,
+                         target_params=None,
+                         poll_until_wait_seconds=None,
+                         **kwargs):
     all_target_params = list(target_params) if target_params else []
     if _launcher_params.target_params is not None:
       all_target_params += _launcher_params.target_params
-    new_runner = black_box_cobalt_runner.BlackBoxCobaltRunner(
-        launcher_params=_launcher_params,
-        url=url,
-        target_params=all_target_params)
-    return new_runner
+
+    if poll_until_wait_seconds:
+      return black_box_cobalt_runner.BlackBoxCobaltRunner(
+          launcher_params=_launcher_params,
+          url=url,
+          target_params=all_target_params,
+          poll_until_wait_seconds=poll_until_wait_seconds,
```

Comment:
This seems like a pass-through here for `poll_until_wait_seconds`, i'd recommend just leaving it in "kwargs". E.g. only `else` part of this block should be needed and you should be able to drop `poll_until_wait_seconds` named arg from the function here.